terraform/testcases/prometheus_sd_awsprw/otconfig.tpl

```diff
@@ -0,0 +1,31 @@
+receivers:
+  prometheus:
+    config:
+      scrape_configs:
+      - job_name: 'kubernetes-service-endpoints'
+        kubernetes_sd_configs:
+        - role: endpoints
+        relabel_configs:
+        - source_labels: [ __meta_kubernetes_namespace ]
+          action: keep
+          regex: "aoc-ns-${testing_id}"
+        - source_labels: [ __meta_kubernetes_pod_label_app ]
+          action: keep
+          regex: "sample-app"
+exporters:
+  awsprometheusremotewrite:
+    endpoint: ${cortex_instance_endpoint}/api/v1/remote_write
+    aws_auth:
+      region: ${region}
+      service: "aps"
```

Comment:
Should this use prometheus remote write and sigv4 instead? Or is this test specifically for `awsprw`?

Reply:
any of the tests with the `_awsprw` suffix are the same as tests that already exist in the repository